### ISSUE TITLE
Planner needs a new icon

### ISSUE THREAD

**Issue by alainm23** [alainm23/planner#1]
The current icon does not reflect what Planner wants to do

**Comment by ArthurS1** [comment]
What do you think about something like that?
![builder 64](https://user-images.githubusercontent.com/33732772/38993031-bf8ef61e-43e2-11e8-904c-78cda9cc7660.png)

**Comment by alainm23** [comment]
Hey, thanks, the model looks good, although I already saw several applications using the same icon. We need Planner to have a different icon, easy to distinguish and novel (especially the latter)

**Comment by Alecaddd** [comment]
I can help you with that if you want. I did Sequeler, Akira, and Lights-up icons.
Do you have some ideas and guidelines in terms of what would you like to have?

**Comment by alainm23** [comment]
I want your opinion @Alecaddd This was my first problem, since I created my icon by grabbing models of other icons and putting them together. But the time has passed and I feel that I'm an icon if it looks good.
What do you think of the icon, really the icon of a hammer reflects planning or working on something?

**Comment by alainm23** [comment]
I think the current icon is fine, although any improvement will be well received.

**Comment by ArthurS1** [comment]
I think maybe just adding something near the hammer would be nice, just so that people dont mistake this app with some kind of developpement tool

**Comment by ArthurS1** [comment]
Something like this maybe, or replacing the calendar with a clipboard 
![68747470733a2f2f63646e2e7261776769742e636f6d2f616c61696e6d32332f706c616e6e65722f6d61737465722f646174612f69636f6e732f3132382f636f6d2e6769746875622e616c61696e6d32332e706c616e6e65722e737667 svg](https://user-images.githubusercontent.com/33732772/39240698-e49a13fc-4884-11e8-95d0-1f4947243495.png)

**Comment by alainm23** [comment]
What do you think of this icon
![com github alainm23 planner svg](https://user-images.githubusercontent.com/33765137/39835238-30c68dc6-5395-11e8-84fe-5b1043216d30.png)

**Comment by Alecaddd** [comment]
Here's my proposal for the icon.
Making it a bit more unique and eccentric.
If you're using this thread to list bugs or let other help you, should reopen the issue.
Is not a bad thing to have issues opened until they're completely solved. It's also a good practice to let other users or possible contributors know where the can help.

![com github alainm23 planner](https://user-images.githubusercontent.com/2527103/39911862-b5a257d8-54b1-11e8-8b91-97c8ad86a3c6.png)

**Comment by alainm23** [comment]
It looks very good, although the truth does not manage to understand the concept at first sight.

**Comment by Alecaddd** [comment]
I agree that it's more an abstract concept, but it communicates some uniqueness and peculiarity that resonates in the app.
The blue shape resembles the letter P, but also looks like the face of a stylized and cute animal.
The "teeth" are the typical colored labels of a paper planner.
Anyway, here's the link to the SVG if you wanna try how it looks on the app drawer or dock, or if you wanna use it to edit it. Feel free to toss it in the trash bin if you don't like :smile: 
https://file.io/60It9U

**Comment by ArthurS1** [comment]
I have to agree, this icon is really cool :)

**Comment by alainm23** [comment]
@Alecaddd  Sorry but it gives me: Page not found

**Comment by Alecaddd** [comment]
@alainm23 Uploaded to another host: https://ufile.io/7mpdx

**Comment by Hedronmx** [comment]
Que opinas?
![rojo n](https://user-images.githubusercontent.com/26935602/40693896-ba59b5fc-636e-11e8-9a07-e615b774edb7.png)
![rojo](https://user-images.githubusercontent.com/26935602/40693897-ba6c4852-636e-11e8-9354-7a54547d1504.png)
![azul blanco](https://user-images.githubusercontent.com/26935602/40693898-ba7dcde8-636e-11e8-854e-98c66d891a05.png)
![azul](https://user-images.githubusercontent.com/26935602/40693899-ba8ffcf2-636e-11e8-8fe1-81005a65bdf9.png)
